calico-enterprise/threat/suspicious-domains.mdx

```diff
@@ -121,10 +121,10 @@ Use the push method if your threat feeds that are not in newline-delimited forma
    kubectl apply -f <your_threatfeed_filename>
    ```
 
-3. Configure or program your threat feed to write updates to Elasticsearch. This Elasticsearch document is in the index **.tigera.domainnameset.\<cluster_name\>** and must have the ID set to the name of the global threat feed object. The doc should have a single field called **domains**, containing a list of domain names. For example:
+3. Configure or program your threat feed to write updates to Elasticsearch. This Elasticsearch document is written using alias **tigera_secure_ee_threatfeeds_domainnameset.\<cluster_name\>.** and must have the ID set to the name of the global threat feed object. The doc should have a single field called **domains**, containing a list of domain names. For example:
```

Comment:
This Elasticsearch document is written using the **tigera_secure_ee_threatfeeds_domainnameset.\<cluster_name\>.** alias and ...